Annotation/declarative_golden_set.json

```diff
@@ -317,7 +341,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "that such thinking forces us to speak of ourselves in terms of \u201cwhat we have\u201d instead of \u201cwho we are\u201d (http://www.firstthings.com/article.php3?id_article=6047)",

```

Comment:
I would delete the brackets with link, I'm not sure we would say this is an adjunct in the grammar?